src/main/java/wooteco/subway/line/dao/LineDaoJdbcTemplate.java

```diff
@@ -0,0 +1,79 @@
+package wooteco.subway.line.dao;
+
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.jdbc.core.simple.SimpleJdbcInsert;
+import org.springframework.stereotype.Repository;
+import wooteco.subway.line.Line;
+
+import javax.sql.DataSource;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+@Repository
+public class LineDaoJdbcTemplate implements LineDao {
```

Comment:
```suggestion
public class JdbcLineDao implements LineDao {
```
보통 세부 구현체를 앞에 명시해줍니다.

Reply:
😁😁😁 수정하였습니다 ><